CHANGELOG.md

```diff
@@ -1,12 +1,45 @@
-# 12.0.0-rc.3 "resin-ensign" (2021-05-11)
+# 12.0.0 "azurite-insight" (2021-05-12)
+
+## Highlights
+
+Sass theming API has been reworked so that clients can take advantage of `@use`. This includes:
+* A single entry point into `@angular/material` and `@angular/cdk`
+* Renamed functions, mixins, and variables to be a better reflection of what they do
+
```

Comment:
I would add a note here that calls out that applications using `node-sass` will need to switch to the `sass` package